pages/features/internationalization.mdx

```diff
@@ -0,0 +1,43 @@
+import { ImageTabs } from '../../components/ImageTabs';
+
+Start UI already implements internationalization, which means that it is designed to support multiple languages.
+It currently supports French, English, Arabic, and Swahili, and is set up to display text in these languages in a way that is appropriate
```

Comment:
It currently... and Swahili, which is a good set to show the possibilities, and is .......

Reply:
I think the "which is a good set to show the possibilities" part is way overkill. What makes a set "good" here ?